TKeazirian.HTTPServer.Tests/Router/RouterTests.cs

```diff
@@ -166,41 +166,23 @@ public void IfOptionsMethodIsAllowedOnEndpointThenGivesAppropriateHeaders()
         Assert.Empty(response.GetBody());
     }
 
-    [Fact]
-    public void PostForSimpleOptionsPosts()
-    {
-        Request testRequest = new Request("POST", "/method_options2", "", "Mock body");
-        var testRoutesConfig = new RoutesConfig(new Dictionary<string, Handler>
-        {
-            { "/test_path", new MockHandler() },
-            { "/method_options2", new MockPostHandler() }
-        });
-        Router router = new Router(testRoutesConfig);
-
-        string testHeaders = HelperFunctions.CreateTestResponseHeaders("Mock body");
-
-        Response response = router.Route(testRequest);
-
-        Assert.Equal("HTTP/1.1 200 OK\r\n", response.ResponseStatusLine);
-        Assert.Equal(testHeaders, response.ResponseHeaders);
-        Assert.Equal("Mock body", response.ResponseBody);
-    }
-
-    [Fact]
-    public void PutForSimpleOptionsReturns501NotImplemented()
+    [Theory]
+    [InlineData("POST")]
+    [InlineData("PUT")]
+    public void PostForSimpleOptionsReturns501NotImplemented(string method)
     {
-        Request testRequest = new Request("PUT", "/method_options2", "", "Mock body");
+        Request testRequest = new Request(method, "/method_options2", "", "Mock body");
         var testRoutesConfig = new RoutesConfig(new Dictionary<string, Handler>
         {
-            { "/test_path", new MockHandler() },
             { "/method_options2", new MockPostHandler() }
         });
```

Comment:
Should this be updated to some generic endpoint? to avoid confusing with you actual implementation.  Looks like '/test_endpoint' is what you are using above.

Reply:
Fixed!